protos/getstrm/pace/api/entities/v1alpha/entities.proto

```diff
@@ -9,6 +9,11 @@ option go_package = "github.com/getstrm/pace/protos/api/entities;entities";
 option java_multiple_files = true;
 option java_package = "com.getstrm.pace.api.entities.v1alpha";
 
+message DataResourceRef {
+  entities.v1alpha.ProcessingPlatform  platform = 1;
+  string fqn = 2;
```

Comment:
Please add a comment what `fqn` is. I can imagine that many users do not know what this is. Maybe even a more elaborate name would be wise, as this also is present in the YAML, which not only developers read, but is also intended for other users.